lib/routes/home/widgets/home_drawer/widgets/breez_avatar_dialog.dart

```diff
@@ -144,13 +146,15 @@ class BreezAvatarDialogState extends State<BreezAvatarDialog> {
       final String? userName = nameInputController.text.isNotEmpty
           ? nameInputController.text
           : userProfileCubit.state.profileSettings.name;
-      userProfileCubit.updateProfile(name: userName);
+      await userProfileCubit.updateProfile(name: userName);
+      await webhookCubit.refreshWebhooks();
```

Comment:
I have some concerns here, perhaps we should involve Roy on how the refresh behavior should be.

Should we refresh LNURL automatically every time user changes their username, or should we set it once when user registers?

We will also show the human readable address between the QR code and Copy & Share buttons on `ReceiveLightningAddressPage`. I think setting it once on registration and changing it via an edit button next to the address is the more feasible solution. It currently is not very clear that changing the username updates the users LN Address.

Reply:
The LNURL service is designed to allow a user to change username. It is then reserved for their pubkey until they change it. But yeah, product-wise its up to Roy to decide.

This PR is just the basic logic to set and update the LNURL username, I think any further goes beyond by flutter UI skills